{"team1": {"ubuntu1ping": {"error": "-c: 1: ping: not found\n", "score": 0}, "ubuntu2ping": {"error": "-c: 1: ping: not found\n", "score": 0}, "ubuntu1ssh": {"error": "timed out", "score": 1}, "ubuntu2ssh": {"error": "timed out", "score": 1}, "ubuntu1web": {"error": "HTTPConnectionPool(host='10.0.1.20', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7daa3e7a9a60>, 'Connection to 10.0.1.20 timed out. (connect timeout=20)'))", "score": 0}, "ubuntu2web": {"error": "HTTPConnectionPool(host='10.0.1.30', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7daa3e534440>, 'Connection to 10.0.1.30 timed out. (connect timeout=20)'))", "score": 0}}, "team2": {"ubuntu1ping": {"error": "-c: 1: ping: not found\n", "score": 0}, "ubuntu2ping": {"error": "-c: 1: ping: not found\n", "score": 0}, "ubuntu1ssh": {"error": "[Errno None] Unable to connect to port 22 on 10.0.2.20", "score": 1}, "ubuntu2ssh": {"error": "[Errno None] Unable to connect to port 22 on 10.0.2.30", "score": 1}, "ubuntu1web": {"error": "HTTPConnectionPool(host='10.0.2.20', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7daa3e5366f0>: Failed to establish a new connection: [Errno 113] EHOSTUNREACH'))", "score": 0}, "ubuntu2web": {"error": "HTTPConnectionPool(host='10.0.2.30', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7daa3e537440>: Failed to establish a new connection: [Errno 113] EHOSTUNREACH'))", "score": 0}}}